internal/cli/atlas/backup/restores_list.go

```diff
@@ -60,6 +60,10 @@ func RestoresListBuilder() *cobra.Command {
 		Aliases: []string{"ls"},
 		Short:   "List cloud backup restore jobs for your project and cluster.",
 		Args:    require.ExactArgs(1),
+		Annotations: map[string]string{
+			"args":     "clusterName",
+			"nameDesc": "Name of the Atlas cluster for which you want to retrieve restore jobs.",
```

Comment:
I think the second key needs to match the value of the first pair? @gssbzn 

```suggestion
			"clusterNameDesc": "Name of the Atlas cluster for which you want to retrieve restore jobs.",
```

Reply:
I thought that `nameDesc` has to stay always the same 🤦